prediction_market_agent_tooling/tools/web3_utils.py

```diff
@@ -237,3 +235,15 @@ def send_function_on_contract_tx_using_safe(
     receipt_tx = web3.eth.wait_for_transaction_receipt(tx_hash, timeout=timeout)
     check_tx_receipt(receipt_tx)
     return receipt_tx
+
+
+def send_xdai_to(
```

Comment:
```suggestion
def send_wei_to(
```

Since it sending Weis and not xDais

Reply:
Here Wei is the unit and xDAI is the [native token](https://docs.gnosischain.com/about/tokens/xdai). Hence I believe it's more apropriate to keep xDAI in the function name, and Wei as type in one of the arguments.

Reply:
Fair enough that we work only with xDai.

And the second question here please, how come this doesn't need a private key?

Reply:
I added some comments in the function (which actually only works for testing).
The short version is that `send_transaction` requires that the private_key of the signing account had already been made available to the provider (local_web3), so you don't need to sign manually (web3 handles that for you).
Since we are sending funds only from `default_accounts`, that works, but from other accounts it won't - hence the requirement that this shall only be done for tests (also reflected in the method name).